src/Application/GameConfig.h

```diff
@@ -605,5 +609,14 @@ class GameConfig : public Config {
     };
 
     Window window{ this };
-};
 
+
+    class CheatCommands : public ConfigSection {
+     public:
+        explicit CheatCommands(GameConfig *config);
+
+        std::vector<std::unique_ptr<String>> CommandList;
```

Comment:
Can drop this field.

Reply:
Yes I don't see this being accessible in a straightforward way as the other Config entries. I'll remove it.

Reply:
Just realized something. Who's going to keep them alive then?

Reply:
Just make it private then?